Externals.cfg

```diff
@@ -3,7 +3,7 @@ protocol = git
 repo_url = https://github.com/NOAA-EMC/regional_workflow
 # Specify either a branch name or a hash but not both.
 #branch = develop
-hash = 47d5b4f
+hase = de1220b
```

Comment:
Please test AFTER changes are made to avoid bugs like this!! hase --> hash

Reply:
@danrosen25 Sorry, I don't mean to be pissy!  I've been trying to get one simple test done all day but keep getting interrupted by constant code updates, so got a little frustrated...

Reply:
@gsketefian 
You're right and I apologize. I wanted to move the pointer back to NOAA-EMC before this PR went forward and just eyeballed the change.